### ISSUE TITLE
Ctrl-J in extensions view opens up Chrome downloads page instead of VSCode terminal

### ISSUE THREAD

**Issue by samuela** [cdr/sshcode#151]
Title really says it all. A lot of the time when I hit Ctrl-J, it ends up opening the Chrome "Downloads" page instead of toggling the VSCode terminal on/off.

**Comment by Merith-TK** [comment]
Yeah due to the nature of the software, thare are some keybinds that cant be controlled, also i feel as if this is a issue for normal [Code-Server](https://github.com/cdr/code-server) as this has to deal with code-server and NOT sshcode

**Comment by samuela** [comment]
Oh, that's right. `Ctrl-J` is toggles the bottom bar thingy, but not the terminal per se.

**Comment by Merith-TK** [comment]
Also i cannot reproduce this

**Comment by samuela** [comment]
I'm running 74.0.3729.157 (Official Build) Built on Raspbian , running on Raspbian 10 (32-bit). `sshcode` installed via `go get -u go.coder.com/sshcode`.

**Comment by Merith-TK** [comment]
Hmm, not really, 
I use brave with no issue, but it might be the fact that you are using a Pi?

**Comment by samuela** [comment]
Possibly, I'd imagine it has to do with how the browser shim propagates keyboard events but I can't be sure of that.

**Comment by Merith-TK** [comment]
Unsure as well. But definitely something to look into a bit
